### DISCUSSION TITLE
Bitbucket Cloud Event Source with Multiple Repos

### DISCUSSION THREAD

**Discussion by alexandrud** [argoproj/argo-events#1619] [Q&A]
Hi,

On the bitbucket event source, is it possible to have multiple repositories on the same event source similar to Bitbucket Server and Github?

Looking at the example it seems you need to define a different webhook URL per repo which would basically mean one pod and one ingress per repo. If you have a large number of repos it becomes excessive.

Thank you!

**Comment by dave08** [comment]
See https://github.com/argoproj/argo-events/issues/1831
